build.sbt

```diff
@@ -3,78 +3,97 @@ import scalariform.formatter.preferences._
 
 organization in ThisBuild := "net.ruippeixotog"
 
-scalaVersion in ThisBuild := "2.12.8"
-crossScalaVersions in ThisBuild := Seq("2.11.12", "2.12.8")
+scalaVersion in ThisBuild := "2.13.0"
+crossScalaVersions in ThisBuild := Seq("2.12.8", "2.13.0")
 
-lazy val core = project.in(file("core"))
+lazy val core = project
```

Comment:
Can you please revert all formatting changes done on this PR? I'd prefer if we kept the PR focused only on the addition of support for Scala 2.13.

Reply:
Resolved